src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -225,10 +225,16 @@ internal TypeSymbol CreateErrorType(string name = "")
         /// </summary>
         internal BoundExpression BindValue(ExpressionSyntax node, DiagnosticBag diagnostics, BindValueKind valueKind)
         {
+            // PROTOTYPE(ngafter): Should we have a version of this that does the BindToNaturalType treatment?
```

Comment:
Prototype comment, please address or file a bug.

Reply:
It is addressed (see next method below).  Will delete the comment.

---
In reply to: [301335706](https://github.com/dotnet/roslyn/pull/37052#discussion_r301335706) [](ancestors = 301335706)